scripts/install

```diff
@@ -64,7 +64,8 @@ exec_occ() {
 }
 
 # Set write access for the following commands
-chown -R $app: "$install_dir" "$data_dir"
+chown -R $app:www-data "$install_dir"
+chown -R $app: "$data_dir"
```

Comment:
```suggestion
chown -R $app:www-data "$install_dir"
chmod og-rwx "$install_dir/config"
chown -R $app: "$data_dir"
```

(me being paranoid about compromission of www-data that would result in leaking secrets etc)

Reply:
Idk, do you think it's not enough?

https://github.com/YunoHost-Apps/nextcloud_ynh/blob/30b2d8123a13d315c803486d9abee3607cc2162f/scripts/install#L244

https://github.com/YunoHost-Apps/nextcloud_ynh/blob/30b2d8123a13d315c803486d9abee3607cc2162f/scripts/restore#L88

https://github.com/YunoHost-Apps/nextcloud_ynh/blob/30b2d8123a13d315c803486d9abee3607cc2162f/scripts/upgrade#L330

Reply:
Eeeh chmod 640 will still allow www-data to read the file, so in the event that www-data gets compromised (or some funky path traversal issue is discovered or whatever) it still allows to access secrets